### PR TITLE
Added GitLab_CI sensor.

### PR DESCRIPTION
**Pull request by DanielWinks**
**Description:**
Documentation for a GitLab_CI sensor added in a pull request

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#16561

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

**Comment by DanielWinks** [comment]
`Liquid Exception: undefined method `key?' for nil:NilClass in /opt/build/repo/source/_components/sensor.gitlab_ci.markdown`

Google searches just turn up results for Ruby errors on this message. I based my markdown file on a few other recent components, editing mine as appropriate. I'm unsure what is causing this failure.

File: source/_components/sensor.gitlab_ci.markdown [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+---
+
+The GitLab_CI sensor integrates results reported by CI/CD Pipeline Jobs in [GitLab](https://gitlab.com/).
+
+## {% linkable_title Setup %}
+
+You will need a GitLab repository ID. On the "Details" page for your GitLab repo, just below the project name is "Project ID:"
+Alternatively, you can use "{GitLab_Username}/{GitLab_RepositoryName}", ex: "MyCoolUsername/MyCoolRepository"
+
+You will also need a GitLab API token with at least API permissions. To get a token, they are generated here: [GitLab API Token](https://gitlab.com/profile/personal_access_tokens)
```
Comment (frenck) [inline source/_components/sensor.gitlab_ci.markdown]: Alternative wording suggestion:

>A GitLab token with at least the API permission scope is needed, which can be created on the [GitLab Personal Access Tokens](https://gitlab.com/profile/personal_access_tokens) page of your GitLab User Settings.

File: source/_components/sensor.gitlab_ci.markdown [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+date: 2018-09-10 19:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: gitlab.png
+ha_category: Sensor
+ha_release: 0.79
+ha_iot_class: "Cloud Polling"
+
```
Comment (frenck) [inline source/_components/sensor.gitlab_ci.markdown]: Extra empty line, please remove.

File: source/_components/sensor.gitlab_ci.markdown [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: gitlab_ci
+    gitlab_id: '1234567'
+    token: 'aaaaaaaaaaAAAAAAAAAA'
+```
+
+{% configuration %}
+
```
Comment (frenck) [inline source/_components/sensor.gitlab_ci.markdown]: Extra empty line, please remove.

File: source/_components/sensor.gitlab_ci.markdown [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+  type: string
+token:
+  description: GitLab API Token
+  required: true
+  type: string
+name:
+  description: Sensor name as it appears in Home Assistant. Defaults to 'GitLab CI Status'.
+  required: false
+  type: string
+scan_interval:
```
Comment (frenck) [inline source/_components/sensor.gitlab_ci.markdown]: This is a general option and assumed to be there and not a direct property of the component. So please remove the `scan_interval` option from this list.

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
Hi there @DanielWinks,

Overall, it looks good, just some minor tweaks! Could you please take a look at the review comments above? Awesome 👍

File: source/_components/sensor.gitlab_ci.markdown (hunk 2 above)
Comment (DanielWinks) [reply source/_components/sensor.gitlab_ci.markdown]: Removed.

File: source/_components/sensor.gitlab_ci.markdown (hunk 4 above)
Comment (DanielWinks) [reply source/_components/sensor.gitlab_ci.markdown]: Removed.

File: source/_components/sensor.gitlab_ci.markdown (hunk 3 above)
Comment (DanielWinks) [reply source/_components/sensor.gitlab_ci.markdown]: Removed.

File: source/_components/sensor.gitlab_ci.markdown (hunk 1 above)
Comment (DanielWinks) [reply source/_components/sensor.gitlab_ci.markdown]: Your rewording looks better, changed.

**Review by frenck — DISMISSED** [review]
Looks great @DanielWinks! Thanks for updating your PR.
✅ Approved. Can be merged as soon as the parent PR get merged in.

**Review by frenck — APPROVED** [review]

**Pull request merged**